sdk/core/azure-core/src/test/java/com/azure/android/core/internal/util/serializer/DurationSerializerTest.java

```diff
@@ -0,0 +1,74 @@
+package com.azure.android.core.internal.util.serializer;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import org.junit.Test;
+import org.threeten.bp.Duration;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class DurationSerializerTest {
+    @Test
+    public void test_getModule() {
+        SimpleModule module = DurationSerializer.getModule();
+
+        assertNotNull(module);
+    }
+
+    @Test
+    public void serializeDuration() throws IOException {
+        StringBuilderOutputStream outputStream = new StringBuilderOutputStream();
+        JsonGenerator jsonGenerator = new JsonFactory().createGenerator(outputStream);
+        DurationSerializer durationSerializer = new DurationSerializer();
+
+        durationSerializer.serialize(Duration.ofDays(3), jsonGenerator, null);
+        jsonGenerator.flush();
+
+        assertEquals("\"P3D\"", outputStream.toString());
+    }
+
+    @Test
+    public void duration_ofDays_toString() {
+        assertEquals("P5D", DurationSerializer.toString(Duration.ofDays(5)));
```

Comment:
Are there tests for a duration deserializer?

Reply:
No, we currently do not have a `DurationDeserializer` class.